internal/pkg/manifest/svc.go

```diff
@@ -570,3 +572,46 @@ func sortExposedPorts(exposedPorts []ExposedPort) []ExposedPort {
 	})
 	return exposedPorts
 }
+
+// ParseNLBPortMapping returns parsed NLB port and protocol.
+func ParseNLBPortMapping(nlbPort *string) (port *string, protocol *string, err error) {
```

Comment:
Given that the [default when `nil` is TCP](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_PortMapping.html), I wonder if we even need to have a `ParseNLBPortMapping` function or if we can re-use `ParsePortMapping`?

Reply:
I'd agree to what you are saying, but to avoid the duplication of that extra step of checking for protocol is nil and if yes then set as TCP I thought of creating this new method. I can very well use the existing  `ParsePortMapping` and write the if protocol == nil then set it as TCP logic wherever it is required. Does that sound good to you?